rewrite/chrome/app/BUILD.gn.toml

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2025 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+[[plaster]]
+description = "Adding deps to generated_resources grit."
+pattern = '(grit\("generated_resources"\)\s*\{.*?)(\n\})'
```

Comment:
this is still a full regex though? Maybe we should name it `re_pattern` and then later we can add a simplified blob matching as just `pattern`?

Reply:
I will submit these in a separate PR, so we can carry on that discussion on the individual file.